app/presenters/property_presenter.rb

```diff
@@ -40,39 +46,14 @@ def property_value
     property[1]
   end
 
-  def render_scripts_table(entry)
-    thead = content_tag(:thead) do
-      content_tag(:tr) do
-        content_tag(:th, 'id').
-          concat(content_tag(:th, 'output'))
-      end
-    end
-    tbody = content_tag(:tbody) do
-      entry[:scripts].map do |script|
-        content_tag(:tr) do
-          content_tag(:td, script[0]).
-            concat content_tag(:td, content_tag(:pre, script[1]))
-        end
-      end.join.html_safe
-    end
-
-    # content_tag(:div,
-    #   content_tag(:table, thead.concat(tbody)),
-    #   class: 'content-textile'
-    # )
-    content_tag(:table, thead.concat(tbody), class: 'table table-condensed')
-  end
-
   def render_table
     values         = property_value
-    column_names   = values.map(&:keys).flatten.uniq.sort.map(&:to_sym)
-    sorted_entries = column_names.include?(:port) ? values.sort_by{ |h| h[:port] } : values
-
-    output = if column_names.delete(:scripts)
-               render_tabs(sorted_entries)
-             else
-               ''
-             end
+    column_names   = values.map(&:keys).flatten.uniq.sort.map(&:to_sym).
+                     delete_if{ |cn| !services_table_columns.include?(cn) }
+    table          = table_info(
+      values: values,
+      sort_by_port: column_names.include?(:port)
```

Comment:
Likewise, this line can be removed if we're not dealing with the 'services' table.